pyartifactory/objects.py

```diff
@@ -64,7 +64,11 @@ class Artifactory:
     """Models artifactory."""
 
     def __init__(
-        self, url: str, auth: Tuple[str, str], verify: bool = True, cert: str = None,
+        self,
+        url: str,
+        auth: Tuple[str, SecretStr],
```

Comment:
I'm not sure about this, now users need to specifically pass a SecretStr when instantiating the lib or else they will have a type error. I think I would prefer if we kept this typed as `Tuple[str, str]`, and either we cast manually inside or we just ignore this error.